Add New Featured Artist: Ritorikal news post (#8048)
ppy/osu-wiki: news/2022-09-24-new-featured-artist-ritorikal.md

---
layout: post
title: "New Featured Artist: Ritorikal"
date: 2022-09-24 10:00:00 +0000
---

We're proud to welcome **Ritorikal** to our Featured Artist collection!

![](https://assets.ppy.sh/artists/305/header.jpg)

Merging various influences, genres, and his own unending curiosity, producer **Ritorikal** has been generating tunes since as far back as 2009. While electronic music is his shtick, he tends to pull in more organic sounds on top (primarily strings, keys, and guitars), making for a less conventional EDM vibe.

> Oh, and **Ritorikal** is also [an osu! player](https://osu.ppy.sh/users/8061786), which miiiight relate to a few of his influences like *MitiS*, *Seven Lions*, [*Koven*](https://osu.ppy.sh/beatmaps/artists/293), and *Virtual Riot*.

Reviewer: The "which might relate to" doesn't make much sense to me. I don't know what it's trying to say.

Author: I've slapped a reword on this to better articulate the (probable) intent of it, at least as far as I understand it.

    </video>
</div>

### Ritorikal - Synergy

Check out [the map from the video above](https://osu.ppy.sh/beatmapsets/1742868) hosted by [Hinsvar](https://osu.ppy.sh/users/1249323)!

<audio controls>
    <source src="https://assets.ppy.sh/artists/305/Synergy/Ritorikal%20-%20Synergy.mp3" type="audio/mpeg">
</audio>

### Ritorikal - Jive

<audio controls>
    <source src="https://assets.ppy.sh/artists/305/Jive/Ritorikal%20-%20Jive.mp3" type="audio/mpeg">
</audio>

### D I V I N I T Y - Our Final Goodbyes (feat. Leorinda & Hatim) (Ritorikal Remix)

<audio controls>
    <source src="https://assets.ppy.sh/artists/305/Our%20Final%20Goodbyes%2FD%20I%20V%20I%20N%20I%20T%20Y%20-%20Our%20Final%20Goodbyes%20(feat.%20Leorinda%20%26%20Hatim)%20(Ritorikal%20Remix).mp3" type="audio/mpeg">
</audio>

### Ritorikal - Imagine

<audio controls>
    <source src="https://assets.ppy.sh/artists/305/Imagine/Ritorikal%20-%20Imagine.mp3" type="audio/mpeg">
</audio>

---

Mappers need music to create beatmaps. [**Ritorikal**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/305) has music. You're a mapper. You can put two and two together.

> One more artist is coming this week, and they're a classic you've been anticipating for years. Stalk the [@osugame Twitter](https://twitter.com/osugame) this coming Saturday for all the juice.

Reviewer 2: ```suggestion
One more artist is coming this week, and they're a classic you've been anticipating for years. Stalk the [@osugame Twitter](https://twitter.com/osugame) this coming Wednesday for all the juice.
```

Reviewer 2: ...actually shouldn't this be something like

```suggestion
Our next artist is a classic you've been anticipating for years. Stalk the [@osugame Twitter](https://twitter.com/osugame) this coming Wednesday for all the juice.
```
